gci/views.py

```diff
@@ -14,6 +15,11 @@ def index(request):
     org_id = linked_students[0]['organization_id']
     org_name = linked_students[0]['organization_name']
     s = []
+
+    get_logo()
```

Comment:
Rather bit scary. This is gonna download the images on every single refresh of every single session. 
Since it is to be downloaded only once, it's better checking the disk for availability of image first, if not present, then fetching it.

Another smarter option would be to fetch the images using a custom management command, so that it can be run on every deployment and the images, if changed on GitHub would be updated upon deployment.

Reply:
What do you mean when you say a custom management command?

Reply:
I think we're using django-distill (static site geneeator) exclusively, so it'll only get generated once.

Reply:
@blazeu I believe so - its one of the dependencies